packages/canary/tailwind.config.js

```diff
@@ -82,6 +82,10 @@ export default {
         },
         ai: {
           DEFAULT: 'hsl(var(--ai))'
+        },
+        divergence: {
```

Comment:
This might not be wise to specify divergence as utility decorator. Very few components require custom additions to styles.css and tailwind.config